tool/site/client-react/src/components/prescription-tracker/prescriptionItemTable.tsx

```diff
@@ -31,6 +36,7 @@ export const PrescriptionItems: React.FC<PrescriptionItemsProps> = ({
           <Table.Row>
             <Table.Cell>Identifier</Table.Cell>
             <Table.Cell>Status</Table.Cell>
+            <Table.Cell>Last Dispensed</Table.Cell>
```

Comment:
do we wanna show last dispensed if none of the items have been dispensed?

Reply:
Thought about hiding the column entirely but decided it was better to be consistent

Reply:
i think its neater to hide column but dont mind either way really :+1: looks fine rendered